recipe/meta.yaml

```diff
@@ -1,42 +1,75 @@
 {% set name = "pytorch-forecasting" %}
-{% set version = "0.10.2" %}
+{% set version = "0.10.3" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/p/pytorch_forecasting/pytorch_forecasting-{{ version }}.tar.gz
-  sha256: 069af34c17eea2edf5c11c79931097427b88cef8a862457818a03464260e5534
+  url: https://github.com/jdb78/pytorch-forecasting/archive/refs/tags/v0.10.3.tar.gz
+  sha256: 026a0498cd2444e38bb7db4c998d1dda5ee4c5e6418cce9d825eae06f892a1c6
+  patches:
+    - fix_versions.patch
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  # pyarrow and fastparquet as test dependencies are not available on s390x
+  skip: True  # [py<38 or py>310 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
+  build:
+    - m2-patch  # [win]
+    - patch     # [not win]
   host:
+    # This basically forces to pick scipy (and any other suitable packages in the chain, like numpy) 
+    # that was built with openblas rather than mkl. Openblas typically pairs with llvm-openmp and mkl with intel-openmp on osx-64. 
+    # By default, mkl variants are picked. By picking the variant that uses openblas rather 
+    # than mkl, intel-openmp will not be pulled in and so you will not have llvm-openmp / intel-openmp conflict. 
+    - blas=*=openblas # [osx and x86_64]
     - pip
```

Comment:
don't we need setuptools and wheel ?

Reply:
No, if poetry-core is used then they are redandunt